generate-puppetfile

```diff
@@ -1,29 +1,31 @@
 #!/bin/sh
-MODULEPATH=`mktemp -d`
-LIST=`mktemp`
-for module in $*
-do
+
+MODULEPATH="$(mktemp -d)"
+LIST="$mktemp"
+for module in "$@"; do
   echo "Download $module and its dependencies..."
-  puppet module install $module --modulepath=$MODULEPATH > /dev/null
+  puppet module install "$module" --modulepath="$MODULEPATH" > /dev/null
 done
 
-echo "Module download complete. Generating Puppetfile."
+echo 'Module download complete. Generating Puppetfile.'
 
-puppet module list --modulepath=$MODULEPATH > $LIST
+puppet module list --modulepath="$MODULEPATH" > "$LIST"
 
-sed -i.bak "s/^\/.*/forge 'http:\/\/forge.puppetlabs.com'\n/" $LIST
-sed -i.bak "s/-/\//" $LIST
-sed -i.bak "s/├── /mod '/" $LIST
-sed -i.bak "s/└── /mod '/" $LIST
-sed -i.bak "s/ (/', '/" $LIST
-sed -i.bak "s/v\([[:digit:]]\)/1/" $LIST
```

Comment:
Why does it say /1/ here instead of /\1/, btw? Keeping it that way for now, maybe you had a reason :)